src/black/linegen.py

```diff
@@ -362,6 +365,11 @@ def rhs(line: Line, features: Collection[Feature]) -> Iterator[Line]:
                 line, line_length=mode.line_length, features=features
             )
 
+        # HACK: functions (like rhs) compiled by mypyc don't retain their __name__
```

Comment:
I guess it would be a bigger refactor but I'd like it better if we just stopped using `__name__`.

Reply:
Yeah, I'd prefer doing this later - tbh I'm a bit tired after all of work! I'll open an issue about this.

Reply:
Filed: https://github.com/psf/black/issues/2432.